Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -990,3 +990,36 @@ lemma IsUnramifiedAtInfinitePlaces.card_infinitePlace [NumberField k] [NumberFie
   exact InfinitePlace.isUnramifiedIn K
 
 end InfinitePlace
+
+namespace IsPrimitiveRoot
+
+variable {K : Type*} [Field K] [NumberField K] {ζ : K} {k : ℕ}
+
+theorem nrRealPlaces_eq_zero_of_two_lt (hk : 2 < k) (hζ : IsPrimitiveRoot ζ k) :
+    NumberField.InfinitePlace.NrRealPlaces K = 0 := by
+  refine (@Fintype.card_eq_zero_iff _ (_)).2 ⟨fun ⟨w, hwreal⟩ ↦ ?_⟩
+  rw [NumberField.InfinitePlace.isReal_iff] at hwreal
+  let f := w.embedding
+  have hζ' : IsPrimitiveRoot (f ζ) k := hζ.map_of_injective f.injective
+  have him : (f ζ).im = 0 := by
+    · rw [← Complex.conj_eq_iff_im, ← NumberField.ComplexEmbedding.conjugate_coe_eq]
+      congr
+  have hre : (f ζ).re = 1 ∨ (f ζ).re = -1 := by
+    · rw [← Complex.abs_re_eq_abs] at him
+      have := Complex.norm_eq_one_of_pow_eq_one hζ'.pow_eq_one (by linarith)
+      rw [Complex.norm_eq_abs, ← him] at this
+      by_cases hpos : 0 ≤ (f ζ).re
+      · rw [_root_.abs_of_nonneg hpos] at this
+        left; exact this
+      · rw [_root_.abs_of_neg (not_le.1 hpos)] at this
+        right; simp [← this]
+  cases hre with
```

Comment:
Slight golf
```suggestion
      rwa [Complex.norm_eq_abs, ← him, ← abs_one, abs_eq_abs] at this
```

Reply:
I am sorry, I still don't know how to correctly write a review. I tried to do it from VSCode but that's far from optimal.

Reply:
I just meant you can replace line 1010 by 
```lean
rwa [Complex.norm_eq_abs, ← him, ← abs_one, abs_eq_abs] at this
``` 
and delete the following lines.

Reply:
BTW, I'd be interested to know what people use to write their review since I cannot figure out how to do it properly 😅